src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -361,7 +361,7 @@ def snapToObject(self, lastpoint, active, constrain,
             parent = obj
             subname = self.snapInfo['Component']
         if not obj:
-            self.spoint = cstr(point)
+            self.spoint = self.cstr(point)
```

Comment:
When I was moving the Snapper class and all that stuff, I noticed this `cstr` method and a couple of other broken things, but since nobody complained about it I assumed it was maybe older code that was not used.

I'm glad to see that this is actually fixed now.

Reply:
Great stuffs, i'm going to have to wait for the next official release before I can use it.